apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx

```diff
@@ -260,17 +270,24 @@ const useChangedFieldMessagesOf = (ticketChange) => {
             )
         }
 
-        if (ticketChange.sender.fingerprint === 'auto-close') {
-            return field === 'statusDisplayName' && AutoClosedMessage
-        }
-
         const valueFrom = ticketChange[`${field}From`]
         const valueTo = ticketChange[`${field}To`]
         const isValueFromNotEmpty = !isNil(valueFrom)
         const isValueToNotEmpty = !isNil(valueTo)
         const formattedValueFrom = formatField(field, valueFrom, TicketChangeFieldMessageType.From)
         const formattedValueTo = formatField(field, valueTo, TicketChangeFieldMessageType.To)
 
+        if (isAutoCloseTicketChanges(ticketChange)) {
+            return (
+                <FormattedMessage
```

Comment:
we use another pattern to formatted messages. Don't use FormattedMessage tag

Reply:
refactored it